docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -136,6 +136,42 @@ The database connection tab shows the command that can be used to connect to the
 modify the database name of the connection and then click the Run button to open a new terminal tab
 with that command executed.
 
+## Connecting to an application
+
+Teleport Connect supports launching HTTP applications in the browser, as well as creating
+authenticated tunnels for HTTP and TCP applications. Access to AWS or SAML applications is currently
+not supported – please use tsh or the Web UI instead.
```

Comment:
If support for AWS apps lands on master (https://github.com/gravitational/teleport/pull/37323), we should change this to
>Access to SAML applications is currently not supported – please use Web UI instead. 

and slightly expand the "Launching an application in the browser" section with selecting an IAM role.